app/controllers/challenge_assignments_controller.rb

```diff
@@ -33,7 +33,7 @@ def load_assignment_from_id
   def owner_only
     return if current_user == @challenge_assignment.offering_pseud.user
 
-    flash[:error] = t("challenge_assignments.not_owner", default: "You aren't the owner of that assignment.")
+    flash[:error] = t(".validations.not_owner", default: "You aren't the owner of that assignment.")
```

Comment:
For this and the other changed translations, the `default` argument should be removed because the translation is now in the locale file. Also, it would be nice to remove the keys from `ignore_missing` in `config/i18n-tasks.yml`.